CMakeLists.txt

```diff
@@ -316,9 +332,11 @@ endif()
 
 if (MINGW)
     target_compile_definitions(${PROJECT} PRIVATE _WIN32_WINNT=0x0501 SDL_MAIN_HANDLED=1)
+    # someone at yaml-cpp forgot to expose this as a target definition:
+    # https://github.com/jbeder/yaml-cpp/blob/c73ee34704c512ebe915b283645aefa9f424a22f/CMakeLists.txt#L49
+    target_compile_definitions(${PROJECT} PRIVATE YAML_CPP_STATIC_DEFINE)
```

Comment:
It might be worth upstreaming this change? No blocker for now though, of course.

Reply:
https://github.com/jbeder/yaml-cpp/issues/1116
https://github.com/jbeder/yaml-cpp/issues/1117